ee-9/source-transform/pom.xml

```diff
@@ -59,6 +59,7 @@
         <module>jsf/subsystem</module>
         <module>mail</module>
         <module>microprofile/reactive-streams-operators-smallrye/cdi-provider</module>
+        <module>microprofile/health-smallrye</module>
```

Comment:
This *was* ok but a couple other PRs have https://github.com/wildfly/wildfly/pull/15176/files#diff-1375aa050fa94d63cc4058a8fbdd07c42da879866e881c4e4ec2ed1f0c4983daR512 in it which will introduce conflict. I'll likely end up merging one of those today.